static/js/timerender.js

```diff
@@ -40,6 +40,39 @@ exports.render_now = function (time) {
     return [time.toString("MMM\xa0dd"), false];
 };
 
+// Current date is passed as an argument for unit testing
+exports.last_seen_status_from_date = function (last_active_date, current_date) {
+    if (typeof  current_date === 'undefined') {
+         current_date = new XDate();
+    }
+    var start_of_today = set_to_start_of_day(current_date.clone());
+    var start_of_other_day = set_to_start_of_day(last_active_date.clone());
+    var days_old = Math.round(start_of_other_day.diffDays(start_of_today));
+    if (days_old === 0) {
```

Comment:
I think we should consider the change shown below.  If it's just after midnight for me, I'm interested to know that Alice was on 10 hours ago and Bob was on 14 hours ago.  I feel like that level of specificity is helpful to the user without being too detailed.

Regardless of what we decide here, we should definitely write tests for these scenarios:
* User was present at 1159 yesterday, and it's just past midnight for me (i.e. the current date).
* User was present at 1800 (6:00pm) yesterday.
* User was present at 2359 (11:59) yesterday.

pinging @timabbott to get his thoughts on what the desired behavior here should be.

~~~ js
var hours_old = last_active_date.diffHours(currentDate);

if (hours_old < 24)
    // do hours logic, and you always return here...might
    // even be worth extracting a small function
}   

var days_old = Math.round(hours_old / 24.0);

if (days_old === 1) {
    // ...
}